Fix sentence fragment (#26022)
dotnet/AspNetCore.Docs: aspnetcore/performance/caching/memory.md

:::code language="csharp" source="memory/samples/6.x/CachingMemorySample/Snippets/Pages/Index.cshtml.cs" id="snippet_OnGetCacheGetOrCreateAbsolute" highlight="5":::

> A cached item set with a sliding expiration is only at risk of becoming stale. If the cached item is repeatedly accessed within the sliding expiration interval never reaching the end of an internal, the item never expires. Combine a sliding expiration with an absolute expiration to guarantee that the item expires when its absolute expiration time is reached. The absolute expiration sets an upper bound on how long the item can be cached while still allowing the item to expire earlier if it isn't requested within the sliding expiration interval. If either the sliding expiration interval *or* the absolute expiration time pass, the item is evicted from the cache.

Author: ```suggestion
A cached item set with only a sliding expiration is at risk of never expiring. If the cached item is repeatedly accessed within the sliding expiration interval, the item never expires. Combine a sliding expiration with an absolute expiration to guarantee the item expires. The absolute expiration sets an upper bound on how long the item can be cached while still allowing the item to expire earlier if it isn't requested within the sliding expiration interval. If either the sliding expiration interval *or* the absolute expiration time pass, the item is evicted from the cache.
```

Author: I removed "stale" as its not used anywhere else in the page so it may be confusing: stale could mean "out-of-date with the real value" or something. "never expiring" is more clear in my mind.

> A cached item set with a sliding expiration only is at risk of becoming stale. If it's accessed within the sliding expiration interval, the item never expires. Combine a sliding expiration with an absolute expiration to guarantee that the item expires once its absolute expiration time passes. The absolute expiration sets an upper bound to how long the item can be cached while still allowing the item to expire earlier if it isn't requested within the sliding expiration interval. When both absolute and sliding expiration are specified, the expirations are logically ORed. If either the sliding expiration interval *or* the absolute expiration time pass, the item is evicted from the cache.

Reviewer: Something like that! 😄 .... I'm off to 🛌💤. *No doubt Merlin the 😸 will have me up at 4am!*

> A cached item set with a sliding expiration only is at risk of becoming stale. If it's accessed within the sliding expiration interval, the item never expires. Combine a sliding expiration with an absolute expiration to guarantee that the item expires once its absolute expiration time passes. The absolute expiration sets an upper bound to how long the item can be cached while still allowing the item to expire earlier if it isn't requested within the sliding expiration interval. When both absolute and sliding expiration are specified, the expirations are logically ORed. If either the sliding expiration interval *or* the absolute expiration time pass, the item is evicted from the cache.

Author: It should be "never reaching the end of an **interval**", not "internal".
